tests/tts_tests/test_tacotron2_train_fsspec_path.py

```diff
@@ -34,8 +34,8 @@
 
 # train the model for one epoch
 command_train = (
-    f"CUDA_VISIBLE_DEVICES='{get_device_id()}' python TTS/bin/train_tts.py --config_path file://{config_path} "
```

Comment:
when you remove ```file://``` then it is not different than a regular training test. Am I wrong?

Reply:
On my system the path was incorrectly parsed, it created a folder `file:` 
![image](https://user-images.githubusercontent.com/17219561/160111397-02fa5552-59b5-443e-9039-888befc9ebd7.png)
I'm not sure what's the expected behavior, I just thought it was a bug.
I can reverse the commit, it was just to trigger the CI since it was bugged yesterday.

Reply:
Because we removed fsspec saving in the trainer since it broke the checkpoints uploaded to the cloud. I think we can even remove the test completely